test/unit/models/tasks.test.js

```diff
@@ -332,8 +332,8 @@ describe("tasks", function () {
 
     it("Should update task status COMPLETED to DONE", async function () {
       const res = await tasks.updateTaskStatus();
-      expect(res.totalTasks).to.be.equal(8);
-      expect(res.totalUpdatedStatus).to.be.equal(8);
+      expect(res.totalTasks).to.be.equal(12);
+      expect(res.totalUpdatedStatus).to.be.equal(12);
```

Comment:
why this change?

Reply:
I increased the tasks in the mock data to include the required data for the new API. This addition increased the task count by 4. While verifying the test cases, I noticed some were failing due to the changes in mock data, so I updated and fixed them accordingly.